components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.edit/src/org/wso2/integrationstudio/gmf/esb/provider/CallTemplateParameterItemProvider.java

```diff
@@ -190,6 +192,12 @@ public String getText(Object object) {
 
         if (parameterType.equalsIgnoreCase(RuleOptionType.VALUE.getName())) {
             if (parameterValue != null) {
+            	String operatingSystem = System.getProperty("os.name");
+            	if (operatingSystem != null && operatingSystem.toLowerCase().contains(OPERATING_SYSTEM_WINDOWS)) {
+            		return parameterName == null || parameterName.length() == 0
+                            ? getString("_UI_CallTemplateParameter_type")
+                            :parameterNameLabel + NAME_VALUE_SEPERATOR + parameterValue;
```

Comment:
A space is required in between : and parameterNameLabel

Reply:
Fixing in : [1123](https://github.com/wso2/integration-studio/pull/1123)